gdl_hyperopt_HPC.py

```diff
@@ -0,0 +1,163 @@
+"""Hyperparamater optimization for GDL using hyperopt
+
+This is a template for using hyperopt with GDL. The my_space variable currently needs to
+be modified here, as well as GDL config modification logic within the objective_with_args
+function.
+
+"""
```

Comment:
Can you put a section in the `README.md` on how to launch it pls :)

Reply:
will be removed